src/app/services/baw-api/account.service.ts

```diff
@@ -36,3 +37,25 @@ export class AccountService extends ReadonlyApi<User, []> {
     return this.apiShow(endpoint(model, Empty));
   }
 }
+
+@Injectable({
+  providedIn: "root"
+})
+export class AccountResolver extends ListResolver<User> {
+  constructor(api: AccountService) {
+    super(api, () => []);
+  }
+}
+
+@Injectable({
+  providedIn: "root"
+})
+export class ShallowSiteResolver extends ShowResolver<User> {
+  constructor(api: AccountService) {
+    super(
+      api,
+      params => parseInt(params.get("accountId"), 10),
+      () => []
+    );
+  }
```

Comment:
Okay, that's a lot of ceremony still.

I'd like to see this reduced to a function. On usage something like `ListResolver<User>(AccountsService)`